develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -62,6 +62,36 @@ specific constant values, such as web keys
 </ul>
 </p>
 
+<p style="margin-left: 40px;">
+Not all components have been migrated from the portal-service API. Their API
```

Comment:
I know I'm reviewing this a bit late, but I happened to see this and wanted to ask "what does this mean?" I've read this paragraph several times but I'm not sure what we are trying to convey here. Could you clarify?